### ISSUE TITLE
Stuck on vedeo upload

### ISSUE THREAD

**Issue by Bardiamist** [onfido/onfido-android-sdk#119]
#### What was the expected behaviour?
Onfido should upload video successfully and go to Finish step.

#### What happened instead?
App stuck with infinite loader. I waited 3 min. I can't close Onfido. In Android Studio I see `HTTP FAILED: java.IO.IOException: Canceled`

#### Version info:

- `onfido-capture-sdk` version: 5.6.0
- Android API version(s): 27

- `compileSdkVersion`: 28
- `targetSdkVersion`: 28
- `minSdkVersion`: 21

#### Steps to reproduce:
Try to upload video.

#### Onfido Flow initialization code:
``` Java
package com.valr.app;

import android.app.Activity;
import android.content.Intent;

import androidx.annotation.NonNull;

import com.facebook.react.bridge.ActivityEventListener;
import com.facebook.react.bridge.BaseActivityEventListener;
import com.facebook.react.bridge.ReactApplicationContext;
import com.facebook.react.bridge.ReactContextBaseJavaModule;
import com.facebook.react.bridge.ReactMethod;
import com.facebook.react.bridge.Callback;
import com.onfido.android.sdk.capture.Onfido;
import com.onfido.android.sdk.capture.ExitCode;
import com.onfido.android.sdk.capture.OnfidoConfig;
import com.onfido.android.sdk.capture.OnfidoFactory;
import com.onfido.android.sdk.capture.errors.OnfidoException;
import com.onfido.android.sdk.capture.ui.camera.face.stepbuilder.FaceCaptureStepBuilder;
import com.onfido.android.sdk.capture.ui.options.FlowStep;
import com.onfido.android.sdk.capture.upload.Captures;

public class OnfidoSDKModule extends ReactContextBaseJavaModule {

    private static final String E_ACTIVITY_DOES_NOT_EXIST = "E_ACTIVITY_DOES_NOT_EXIST";
    private final Onfido client;
    private String mApplicantId;
    private Callback mSuccessCallback;
    private Callback mErrorCallback;

    private final ActivityEventListener mActivityEventListener = new BaseActivityEventListener() {
        @Override
        public void onActivityResult(final Activity activity, int requestCode, int resultCode, Intent data) {
            super.onActivityResult(activity, requestCode, resultCode, data);
            client.handleActivityResult(resultCode, data, new Onfido.OnfidoResultListener() {
                @Override
                public void userExited(@NonNull ExitCode exitCode) {
                    mErrorCallback.invoke(exitCode.toString());
                }

                @Override
                public void userCompleted(@NonNull Captures captures) {
                    mSuccessCallback.invoke(mApplicantId);
                }

                @Override
                public void onError(OnfidoException exception) {
                    mErrorCallback.invoke(exception.getMessage());
                }
            });
        }
    };

    public OnfidoSDKModule(ReactApplicationContext reactContext) {
        super(reactContext);
        client = OnfidoFactory.create(reactContext).getClient();
        reactContext.addActivityEventListener(mActivityEventListener);
    }

    @Override
    public String getName() {
        return "OnfidoSDK";
    }

    @ReactMethod
    public void startSDK(String token, String applicantId, Callback successCallback, Callback errorCallback) {

        Activity currentActivity = getCurrentActivity();
        mApplicantId = applicantId;
        mSuccessCallback = successCallback;
        mErrorCallback = errorCallback;

        if (currentActivity == null) {
            mErrorCallback.invoke(E_ACTIVITY_DOES_NOT_EXIST);
            return;
        }

        FlowStep faceCaptureStep = FaceCaptureStepBuilder.forVideo()
                .build();

        final FlowStep[] flowStepsWithOptions = new FlowStep[]{
                FlowStep.CAPTURE_DOCUMENT,
                faceCaptureStep,
                FlowStep.FINAL
        };

        try {
            OnfidoConfig onfidoConfig = OnfidoConfig.builder(currentActivity)
                    .withCustomFlow(flowStepsWithOptions)
                    .withSDKToken(token)
                    .build();
            client.startActivityForResult(currentActivity, 1, onfidoConfig);
        }
        catch (Exception e) {
            mErrorCallback.invoke(e.toString());
            mErrorCallback = null;
        }
    }
}

```

#### Issue severity information (number of users/crashes if applicable):
That problem happened on my first try and not reproduced on next two times.

#### Additional LogCat output:
[Logs](https://imgur.com/GlGf9yd)

#### Other useful info:
_screenshots, videos, etc._

**Comment by gokhanaliccii** [comment]
@Bardiamist Thanks for reporting the problem. We've managed to reproduce the problem on our side. We'll update the issue once we fix the error. 

Thanks for your patience

**Comment by burakeregar** [comment]
This issue has been fixed as part of version [7.1.0](https://github.com/onfido/onfido-android-sdk/releases/tag/7.1.0)
